docs/Local-Development.md

```diff
@@ -0,0 +1,164 @@
+# Local development
+
+1. [First steps | Local installation](#first-steps)
+1. [Setup testing env](#setup-testing-env)
```

Comment:
Can you increment the numbered items in this list?

Also below under `Install these:`

Reply:
@davidbloss the list doesn't need to be numbered, [the markdown renderer (GitHub) will number them for us.](https://github.com/OpsLevel/kubectl-opslevel/blob/49b41b35b75b926721c2a887f42e56f71d432353/docs/Local-Development.md)